src/uu/rm/src/rm.rs

```diff
@@ -100,7 +98,9 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         let options = Options {
             force,
             interactive: {
-                if matches.contains_id(OPT_PROMPT) {
+                if force {
+                    InteractiveMode::Never
+                } else if matches.contains_id(OPT_PROMPT) {
```

Comment:
This is a tricky case, because the order matters here:
```
rm -i foo          # prompts
rm -f foo          # does not prompt
rm -if foo         # does not prompt
rm -fi foo         # prompts
```
We should probably let these options override each other.

Reply:
I was wondering about that. Thanks for looking into it

Reply:
Is there a way with matches to figure out which came first other that grabbing the index of each of them and comparing to see which came first?